test/remote/gateways/remote_credorax_test.rb

```diff
@@ -79,6 +85,13 @@ def test_successful_purchase_with_google_pay
     assert_equal 'Succeeded', response.message
   end
 
+  def test_successful_purchase_with_network_token
+    response = @gateway.purchase(@amount, @nt_credit_card, @options)
+    assert_success response
+    assert_equal '1', response.params['H9']
+    assert_equal 'Succeeded', response.message
+  end
+
```

Comment:
💭 Suggestion:

I think that, could be good to add some tests without `source: :network_token`

Reply:
The proposed test would be useless because if payment source is not included in the adapter sources the default value is setted to 'apple_pay'